buildkite/resource_team.go

```diff
@@ -174,7 +227,7 @@ func (t *teamResource) Read(ctx context.Context, req resource.ReadRequest, resp
 		return
 	}
 
-	if teamNode, ok := res.GetNode().(*getNodeNodeTeam); ok {
+	if teamNode, ok := response.GetNode().(*getNodeNodeTeam); ok {
```

Comment:
This also needs the state removal if not found

Reply:
On it 👍